### PR TITLE
New Q&A: Testing the dev version without uninstalling the release one

### PR DESCRIPTION
**Pull request by mlep**
Following https://github.com/JabRef/jabref/issues/6094#issuecomment-597439459
First draft, for review.

### CODE REVIEW

File: en/faq/faqgeneral.md [hunk 1]
```diff
@@ -109,3 +109,7 @@ A: In JabRef 3.0 plugin support was removed, because the development team cannot
 ### Q: Where did the RenameFile plugin go? How do I rename files automatically after importing entries?
 
 A: JabRef does no longer support plugins \(since version 3.0\). Automatic file renaming is now part of the **Quality → Cleanup entries** feature. There you can rename attached files based on the BibTeX key. You can change the format \(pattern\) under **Options → Preferences → Import**, by altering the pattern under "Default PDF file link action".
+
+### Q: How to test a development version without uninstalling the released version?
+
+A: Download the portable version (file JabRef-X.Y.portable_windows.zip, JabRef-X.Y.portable_macos.tar.gz, or JabRef-X.Y.portable_linux.tar.gz), uncompress it, and run the executable.
```
Comment (Siedlerchr) [inline en/faq/faqgeneral.md]:
```suggestion
A: Download the portable version (file JabRef-X.Y.portable_windows.zip, JabRef-X.Y.portable_macos.tar.gz, or JabRef-X.Y.portable_linux.tar.gz), uncompress it to a different folder, and run the executable.
```

**Comment by bernhard-kleine** [comment]
I would rise the question which version you want to prefer. That version should be installed via install and the other one should be started from the portable version.

**Comment by koppor** [comment]
@bernhard-kleine Thank you for the suggestion. I tried to phrase it at https://docs.jabref.org/faq/faqgeneral#q-how-to-test-a-development-version-without-uninstalling-the-released-version.

**Comment by bernhard-kleine** [comment]
It is "portable version" and not "protable".

**Comment by koppor** [comment]
Will fix it.

Did you know that you can also propose a fix?

grafik

**Pull request merged**